about.html

```diff
@@ -0,0 +1,167 @@
+                <a href="#"><a href="#">My Page</a></i></a>
+                <a href="#"><a href="#">Logout</a></i></a>
+            </div>
+        </section>
+
+        <!-------- stat of main-navigation --- -->
+        <nav class="navbar">
+            <div class="container">
+                <div class="nav-logo">
+                    <a href="index.html"><img src="images/nav-img/logo.svg" alt="nav-logo" id="nav-open"></a>
+                </div>
+                <ul class="nav-links flex">
+                    <li><a href="index.html" id="active">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Courses</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <button type="submit" class="btn btn-default">
+                        <a href="#">New Courses!</a>
+                    </button>
+                </ul>
+                <i class="open bx bx-menu"></i>
+                <ul class="mobile-nav">
+                    <i class="close fa fa-times"></i>
+                    <li><a href="index.html" id="active">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#courses">Courses</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                </ul>
+            </div>
+        </nav>
+
+        <!-- ------ start of home section  -->
+        <section class="about-home">
+            <div class="container">
+                <div class="about-home-text">
+                    <h2>Welcome! to Evans code online school</h2>
+                    <h1>GROW YOUR CODING SKILLS TO<br>ADVANCE YOUR CAREER PATH</h1>
+                    <p>Our aim to help you a establish a good fundation in software development. The journey
+                        of becoming a developer can not just be achieved by feeling good behind a complter. We make sure
+                        you gain much understanding of every basic lesson in coding.
+                        Our aim to help you a establish a good fundation in software development. The journey
+                        of becoming a developer can not just be achieved by feeling good behind a complter. We make sure
+                        you gain much understanding of every basic lesson in coding.
+                    </p>
+                    <h3>Starting 05.06.23(MON) ~ (FRI)</h3>
+                    <h6>@ 8am Opo-louker street, West Hills-Weija. Ghana<br>PGS Address-GS-83E4-XXX</h6>
+                    <!-- <a href="#" class="hero-btn">Visit us</a> -->
+                    <p class="home-nl">Please contact us per Email for nay further questions about CC Global Summit
+                        2015!</p>
+                    <div class="mobile">
+                        <a href="#">Explore BootCamp Now!</a>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Please remove unwanted comments from your codebase because it's your codebase make messy if you want to check your previous code then check your commit GitHub History
- [ ] As I mentioned above :point_up: please stick to the Behance design your hero section in the mobile version is not the same as the Behance design :+1: check below :point_down: the comparison between  your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-05-27 00-07-00](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/f4e7a9d0-98db-4014-81b8-7bcbb8890043)


**Behance Design**
![Screenshot from 2023-05-27 00-06-46](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/952fc257-f835-4edc-a202-a641c2fd711a)